plan/README.md

```diff
@@ -0,0 +1,79 @@
+
+## Target Users
+There are two user categories that are expected to make up the bulk of traffic to hitthepolls.com.
+### Trevor - eligible voter for 2020 election
+
+<img src="img/trevor.jpg" width=200>
+
+Trevor is a 30-year-old voter who usually doesn't pay attention to politics. However, with the recent push to have every eligible voter exercise their right to vote, Trevor has felt it is his responsibility to educate himself on who he should vote for based on his aligning opinions or values. Trevor comes to this site seeking information about the election deadlines, and where he should vote. 
+
+### Erica - eligible voter for 2020 election
+
+<img src="img/erica.jpg" width=200>
+
+Erica is a 21-year-old American who usually stays up-to-date on current events and politics. She is in-tune with what has been happening over these last few months leading up to the election. She does struggle with trusting news sources and often uses social media as a news source. She comes to the site looking to learn more about the news sources and how they view things. This will help her get a better understanding of which news sources will be bias towards a certain side or policy. 
+
+## Content
+
+At the bottom of all pages, there will be a survey that will allow the user to inquire about more information regarding the election. Additionally, this survey will ask the user if they feel more educated on the various topics discussed on the site.
+
+### Home
+
+This is where most visitors will land when coming to the site. There will be links to news articles and a countdown to Election Day and other important deadlines.
+
+### About
+
+This page will contain a brief description of the election and will provide basic details about it. Additionally, it will contain links to other sites that can help users learn further about the election and find where their specified voting center is for their hometown. 
+
+### News Sources and Their Biases
+
+This page will have a rundown on popular news sources. For each source, there will be a description about which political party they tend to lean towards and how much they are bias towards that party or ideology. 
+
+### Presidential Candidates
+
+This page will be a comparison of the Presidential candidates and their runningmates or VPs. This comparison will focus on major policies and issues that our country is facing during these times. 
+
+### State and Local Candidates
+
+This page is very similar to previous page. Instead, it will outline the state and local candidates for this election. Since I am from Hamden, Connecticut, I will be focusing on the Connecticut state election and the local election. 
+
+## Similar Sites
+
+### howto.vote
+
+<img src="img/howtovote.png">
+
+### isidewith.com
+
+<img src="img/isidewith.png">
+
+### easyvoterguide.com
+
+<img src="img/easyvoterguide.png">
```

Comment:
What is it that you like about these sites? Will you use any ideas from these sites in your own?